input/resources/au-patient.xml

```diff
@@ -154,7 +154,7 @@
       <path value="Patient.extension"/>
       <sliceName value="recordedSexOrGender"/>
       <short value="Recorded sex or gender type shall be a member of the Common AU Recorded Sex or Gender Type value set if any codes within that value set can apply. Recorded sex or gender source document type shall be a member of the Common AU Recorded Sex or Gender (RSG) Source Document Type value set if any codes within that value set can apply. Recorded sex or gender jurisdiction shall be a member of the Common AU Recorded Sex or Gender (RSG) Source Document Jurisdiction value set if any codes within that value set can apply"/>
-      <comment value="Person Recorded Sex Or Gender type [Common AU Recorded Sex or Gender Type](ValueSet-rsg-type.html) ([extensible](http://hl7.org/fhir/R4/terminologies.html#extensible)). Person Recorded Sex Or Gender source document [Common AU Recorded Sex or Gender (RSG) Source Document Type](ValueSet-rsg-source-document-type.html) ([extensible](http://hl7.org/fhir/R4/terminologies.html#extensible)). Person Recorded Sex Or Gender jurisdiction [Common AU Recorded Sex or Gender (RSG) Source Document Jurisdiction](ValueSet-rsg-source-document-jurisdiction.html) ([extensible](http://hl7.org/fhir/R4/terminologies.html#extensible))."/>
+      <comment value="Person Recorded Sex Or Gender type [Common AU Recorded Sex or Gender Type](https://build.fhir.org/ig/hl7au/au-fhir-base/ValueSet-rsg-type.html) ([extensible](http://hl7.org/fhir/R4/terminologies.html#extensible)). Person Recorded Sex Or Gender source document [Common AU Recorded Sex or Gender (RSG) Source Document Type](https://build.fhir.org/ig/hl7au/au-fhir-base/ValueSet-rsg-source-document-type.html) ([extensible](http://hl7.org/fhir/R4/terminologies.html#extensible)). Person Recorded Sex Or Gender jurisdiction [Common AU Recorded Sex or Gender (RSG) Source Document Jurisdiction](https://build.fhir.org/ig/hl7au/au-fhir-base/ValueSet-rsg-source-document-jurisdiction.html) ([extensible](http://hl7.org/fhir/R4/terminologies.html#extensible))."/>
```

Comment:
Also apply these fixes to:
1. AU Base RelatedPerson
2. AU Base Practitioner

Reply:
:(. Done.